frameworks/cassandra/tests/test_recovery.py

```diff
@@ -27,6 +27,17 @@ def configure_package(configure_universe):
 
 @pytest.mark.sanity
 @sdk_utils.dcos_1_9_or_higher # dcos task exec not supported < 1.9
+def test_node_replace_replaces_seed_node():
```

Comment:
I agree with @NimaVaziri.  In the repro I saw, a series of valid operations against C* resulted in a cluster state it didn't like.  It wasn't a test artifact or readiness check artifact.  C* itself reported that all nodes were `UN` up and normal.  This designation of health appears to have some nuance because despite this report C* was unable to handle the replacement of a node due to an inconsistency in its internal state.